plugin/pxr/cmake/macros/Public.cmake

```diff
@@ -101,7 +101,7 @@ function(pxr_library NAME)
             return()
         endif()
 
-        set(prefix "")
+        set(prefix ${CMAKE_SHARED_LIBRARY_PREFIX})
```

Comment:
We actually want to leave the prefix as an empty string for plugins - otherwise, the maya plugin will be created with at ".lib" prefix on Linux/MacOS